test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -1521,6 +1521,263 @@ public async Task GetTrustResultAsync_WithExpiredRepositoryCertificateAndNoTimes
             }
         }
 
+        [Collection(SigningTestCollection.Name)]
+        public class FallbackFromPrimarySignaturesToCountersignatures
+        {
+            //The settings when validating packages from nuget.org in accept mode (AcceptModeDefaultPolicy + allowUnsigned:false + allowUntrusted:false)
+            private readonly SignedPackageVerifierSettings _defaultNuGetOrgSettings = new SignedPackageVerifierSettings(
+                allowUnsigned: false,
+                allowIllegal: true,
+                allowUntrusted: false,
+                allowIgnoreTimestamp: true,
+                allowMultipleTimestamps: true,
+                allowNoTimestamp: true,
+                allowUnknownRevocation: true,
+                reportUnknownRevocation: false,
+                verificationTarget: VerificationTarget.All,
+                signaturePlacement: SignaturePlacement.Any,
+                repositoryCountersignatureVerificationBehavior: SignatureVerificationBehavior.IfExistsAndIsNecessary,
+                revocationMode: RevocationMode.Online);
+            private readonly SigningTestFixture _fixture;
+            private readonly SignatureTrustAndValidityVerificationProvider _provider;
+
+            public FallbackFromPrimarySignaturesToCountersignatures(SigningTestFixture fixture)
+            {
+                if (fixture == null)
+                {
+                    throw new ArgumentNullException(nameof(fixture));
+                }
+
+                _fixture = fixture;
+                _provider = new SignatureTrustAndValidityVerificationProvider();
+            }
+
+            // Case1: primary signature (trusted + non-expired) doesn't fall back to countersiganture (trusted + non-expired).
```

Comment:
countersiganture -> countersignature

Find and replace all occurrences in this file.

Reply:
Sorry for the typo! Fixed.